tests/parametric/test_tracer_flare.py

```diff
@@ -0,0 +1,147 @@
+    return {
+        "args": {"case_id": "12345", "hostname": "my.hostname", "user_handle": "its.me@datadoghq.com"},
+        "task_type": "tracer_flare",
+    }
+
+
+def _flare_log_level_order() -> Dict[str, Any]:
+    return {
+        "order": [],
+        "internal_order": [
+            "flare-log-level.trace",
+            "flare-log-level.debug",
+            "flare-log-level.info",
+            "flare-log-level.warn",
+            "flare-log-level.error",
+            "flare-log-level.critical",
+            "flare-log-level.off",
+        ],
+    }
+
+
+def _set_log_level(test_agent, log_level: str) -> None:
+    """Helper to create the appropriate "flare-log-level" config in RC for a given log-level.
+    """
+    cfg_id = "flare-log-level.%s" % log_level
+    test_agent.set_remote_config(
+        path="datadog/2/AGENT_CONFIG/%s/config" % cfg_id, payload={"name": cfg_id, "config": {"log_level": log_level}}
+    )
+    test_agent.wait_for_rc_apply_state("AGENT_CONFIG", state=2)
+
+
+def _clear_log_level(test_agent, log_level: str) -> None:
+    """Helper to clear a previously set "flare-log-level" config from RC.
+    """
+    cfg_id = "flare-log-level.%s" % log_level
+    test_agent.set_remote_config(path="datadog/2/AGENT_CONFIG/%s/config" % cfg_id, payload={})
+    test_agent.wait_for_rc_apply_state("AGENT_CONFIG", state=2, clear=True)
+
+
+def _add_task(test_agent, task_config: Dict[str, Any]) -> int:
+    """Helper to create an agent task in RC with the given task arguments.
+    """
+    task_config["uuid"] = uuid4().hex
+    task_id = hash(json.dumps(task_config))
+    test_agent.set_remote_config(path="datadog/2/AGENT_TASK/%s/config" % task_id, payload=task_config)
+    test_agent.wait_for_rc_apply_state("AGENT_TASK", state=2)
+    return task_id
+
+
+def _clear_task(test_agent, task_id) -> None:
+    """Helper to clear a previously created agent task config from RC.
+    """
+    test_agent.set_remote_config(path="datadog/2/AGENT_TASK/%s/config" % task_id, payload={})
+    test_agent.wait_for_rc_apply_state("AGENT_TASK", state=2, clear=True)
+
+
+def trigger_tracer_flare_and_wait(test_agent, task_overrides: Dict[str, Any]) -> Dict:
+    """Creates a "trace_flare" agent task and waits for the tracer flare to be uploaded.
+    """
+    task_config = _tracer_flare_task_config()
+    task_args = task_config["args"]
+    for k, v in task_overrides.items():
+        task_args[k] = v
+
+    task_id = _add_task(test_agent, task_config)
+    tracer_flare = test_agent.wait_for_tracer_flare(task_args["case_id"], clear=True)
+    _clear_task(test_agent, task_id)
+
+    return tracer_flare
+
+
+def assert_valid_zip(content):
+    flare_file = zipfile.ZipFile(BytesIO(b64decode(content)))
+    assert flare_file.testzip() is None, "tracer_file zip must not contain errors"
+    assert flare_file.namelist(), "tracer_file zip must contain at least one entry"
+
+
+@missing_feature(context.library == "cpp", reason="Not implemented")
```

Comment:
Could you use manifest file ?

-> `docs/edit/manifest.yml`

Reply:
yes, I'll make that change

Reply:
ff41ee32cf7abffb4027a9632915ab02dcab3180